crates/core/tedge_mapper/src/c8y/service_monitor.rs

```diff
@@ -13,14 +15,14 @@ pub struct HealthStatus {
     pub status: String,
 }
 
-fn default_type() -> String {
-    "service".to_string()
-}
-
 fn default_status() -> String {
     "unknown".to_string()
 }
 
+fn default_type() -> String {
+    "".to_string()
```

Comment:
```suggestion
    DEFAULT_SERVICE_TYPE.to_string()
```
This seems more appropriate even though I understand that the parsing logic below will convert this empty string to the default type anyway.